api/envoy/admin/v3/config_dump.proto

```diff
@@ -184,6 +191,12 @@ message ClustersConfigDump {
 
     // The timestamp when the Cluster was last updated.
     google.protobuf.Timestamp last_updated = 3;
+
+    // Set if the last update failed, cleared after the next successful update.
```

Comment:
For these new fields, without corresponding Envoy implementation, I'd like to mark them `[#not-implemented-yet:]`.

Reply:
Done.